Orange/widgets/data/owpreprocess.py

```diff
@@ -535,6 +535,112 @@ def createinstance(params):
 # TODO: Model based FS (random forest variable importance, ...), RFE
 # Unsupervised (min variance, constant, ...)??
 
+class RandomFeatureSelectEditor(BaseEditor):
+    #: Strategy
+    Fixed, Percentile, FDR, FPR, FWE = 1, 2, 3, 4, 5
+
+    def __init__(self, parent=None, **kwargs):
+        super().__init__(parent, **kwargs)
+        self.setLayout(QVBoxLayout())
+
+        self.__strategy = RandomFeatureSelectEditor.Fixed
+        self.__k = 10
+        self.__p = 75.0
+
+        box = QGroupBox(title="Strategy", flat=True)
+        self.__group = group = QButtonGroup(self, exclusive=True)
+        self.__spins = {}
+
+        form = QFormLayout()
+        fixedrb = QRadioButton("Fixed", checked=True)
+        group.addButton(fixedrb, RandomFeatureSelectEditor.Fixed)
+        kspin = QSpinBox(
+            minimum=1, value=self.__k,
+            enabled=self.__strategy == RandomFeatureSelectEditor.Fixed
+        )
+        kspin.valueChanged[int].connect(self.setK)
+        kspin.editingFinished.connect(self.edited)
+        self.__spins[RandomFeatureSelectEditor.Fixed] = kspin
+        form.addRow(fixedrb, kspin)
+
+        percrb = QRadioButton("Percentile")
+        group.addButton(percrb, RandomFeatureSelectEditor.Percentile)
+        pspin = QDoubleSpinBox(
+            minimum=0.0, maximum=100.0, singleStep=0.5,
+            value=self.__p, suffix="%",
+            enabled=self.__strategy == RandomFeatureSelectEditor.Percentile
+        )
+        pspin.valueChanged[float].connect(self.setP)
+        pspin.editingFinished.connect(self.edited)
+        self.__spins[RandomFeatureSelectEditor.Percentile] = pspin
+        form.addRow(percrb, pspin)
+
+        # form.addRow(QRadioButton("FDR"), QDoubleSpinBox())
+        # form.addRow(QRadioButton("FPR"), QDoubleSpinBox())
+        # form.addRow(QRadioButton("FWE"), QDoubleSpinBox())
```

Comment:
Commented-out lines in the code are only allowed if preceded by a comment explaining why they are there.